(dp1
S'output'
p2
S"<type 'exceptions.NameError'> global name 'itle_query' is not defined"
p3
sS'layer'
p4
S'C:\\Users\\Ebenezer Appiah\\Desktop\\software\\windows\\web2py\\applications\\rentapp\\controllers/default.py'
p5
sS'code'
p6
S'<code object <module> at 09F82D58, file "C:\\Users\\Ebenezer Appiah\\Desktop\\software\\windows\\web2py\\applications\\rentapp\\controllers/default.py", line 1>'
p7
sS'snapshot'
p8
(dp9
sS'traceback'
p10
S'Traceback (most recent call last):\n  File "C:\\Users\\Ebenezer Appiah\\Desktop\\software\\windows\\web2py\\gluon\\restricted.py", line 219, in restricted\n    exec(ccode, environment)\n  File "C:\\Users\\Ebenezer Appiah\\Desktop\\software\\windows\\web2py\\applications\\rentapp\\controllers/default.py", line 48, in <module>\n  File "C:\\Users\\Ebenezer Appiah\\Desktop\\software\\windows\\web2py\\gluon\\globals.py", line 419, in <lambda>\n    self._caller = lambda f: f()\n  File "C:\\Users\\Ebenezer Appiah\\Desktop\\software\\windows\\web2py\\applications\\rentapp\\controllers/default.py", line 42, in searchresult\n    uploads=db(itle_query).select(orderby=db.uploads.db_location)\nNameError: global name \'itle_query\' is not defined\n'
p11
s.